### PR TITLE
feat(ui): init destop styling foundation

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Init desktop styling

split the styling foundation into mobile.scss and desktop.scss
refactor the page bast layout to suit the desktop & mobile adjustment

image

image

## Linear Issue Reference

log-2398

## Testing

test locally
@logto-io/eng

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/ui/src/components/AppContent/index.tsx [hunk 1]
```diff
@@ -29,9 +29,18 @@ const AppContent = ({ children }: Props) => {
     document.body.classList.add(styles[theme] ?? '');
```
Comment (gao-sun) [inline packages/ui/src/components/AppContent/index.tsx:29]: `remove` and `add` support multiple parameters. may combine them

File: packages/ui/src/components/AppContent/index.tsx [hunk 2]
```diff
@@ -29,9 +29,18 @@ const AppContent = ({ children }: Props) => {
...
   }, [theme]);
 
+  useEffect(() => {
+    if (isMobile) {
+      document.body.classList.add(styles.mobile ?? '');
+
+      return;
+    }
+    document.body.classList.add(styles.desktop ?? '');
```
Comment (gao-sun) [inline packages/ui/src/components/AppContent/index.tsx:38]:
```suggestion
    document.body.classList.add(
      conditionalString(isMobile && styles.mobile),
      conditionalString(!isMobile && styles.desktop)
    );
```

Comment (simeng-li) [reply packages/ui/src/components/AppContent/index.tsx:38]: image

Looks like empty string is not allowed

File: packages/ui/src/components/AppContent/index.tsx (hunk 1 above)
Comment (simeng-li) [reply packages/ui/src/components/AppContent/index.tsx]: updated

File: packages/ui/src/components/AppContent/index.tsx (hunk 2 above)
Comment (simeng-li) [reply packages/ui/src/components/AppContent/index.tsx]: updated using the conditional string however keep the is condition statement

**Pull request merged**